bindings/gumjs/gumdukvalue.c

```diff
@@ -1011,6 +1045,26 @@ _gum_duk_parse_pointer (duk_context * ctx,
     *ptr = GSIZE_TO_POINTER ((gsize) number);
     return TRUE;
   }
+  else if (_gum_duk_is_uint64 (ctx, index, core))
+  {
+    guint64 val = 0;
+    if (_gum_duk_get_uint64 (ctx, index, core, &val))
```

Comment:
This cannot fail as we already checked the type, so the `if` should be removed. Same below.